### PR TITLE
APIV2 implement all GameAdmins routes

### PR DESCRIPTION
**Pull request by TobleroneSwordfish**
[INTERNAL]
## About the PR 
Implements all `/game-admins` routes and creates a couple of missing model types for them.

## Why's this needed? 
I have been sent down the API mines.

### CODE REVIEW

**Review by Wirewraith — APPROVED** [review]

File: code/modules/goonhub/api/requests/admins/AddGameAdminsBody.dm [hunk 1]
```diff
@@ -0,0 +1,34 @@
+/datum/apiBody/admins/post
```
Comment (ZeWaka) [inline code/modules/goonhub/api/requests/admins/AddGameAdminsBody.dm:1]:
```suggestion
/datum/apiBody/gameadmins/post
```
should mirror the spec (`-` is an invalid char tho)

File: code/modules/goonhub/api/requests/admins/UpdateGameAdminsBody.dm [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/datum/apiBody/admins/put
```
Comment (ZeWaka) [inline code/modules/goonhub/api/requests/admins/UpdateGameAdminsBody.dm:1]:
```suggestion
/datum/apiBody/gameadmins/put
```

File: code/modules/goonhub/api/routes/admins/AddGameAdmins.dm [hunk 3]
```diff
@@ -0,0 +1,8 @@
+
+/// POST /game-admins
+/// Add a new game admin
+/datum/apiRoute/admins/post
```
Comment (ZeWaka) [inline code/modules/goonhub/api/routes/admins/AddGameAdmins.dm:4]:
```suggestion
/datum/apiRoute/gameadmins/post
```

File: code/modules/goonhub/api/routes/admins/DeleteGameAdmins.dm [hunk 4]
```diff
@@ -0,0 +1,8 @@
+
+/// DELETE /game-admins/{gameAdmin}
+/// Delete an existing game admin
+/datum/apiRoute/admins/delete
```
Comment (ZeWaka) [inline code/modules/goonhub/api/routes/admins/DeleteGameAdmins.dm:4]:
```suggestion
/datum/apiRoute/gameadmins/delete
```

File: code/modules/goonhub/api/routes/admins/GetGameAdmins.dm [hunk 5]
```diff
@@ -0,0 +1,8 @@
+
+/// GET /game-admins
+/// List paginated and filtered game admins
+/datum/apiRoute/admins/get
+	method = RUSTG_HTTP_METHOD_GET
+	path = "/game-admins"
+	queryParams = list("filters", "sort_by", "descending", "per_page") // string, string, string, string
```
Comment (ZeWaka) [inline code/modules/goonhub/api/routes/admins/GetGameAdmins.dm:7]:
```suggestion
	queryParams = list("filters", "sort_by", "descending", "per_page") // string[], string, string, string
```

File: goonstation.dme [hunk 6]
```diff
@@ -1155,8 +1157,14 @@ var/datum/preMapLoad/preMapLoad = new
...
 #include "code\modules\goonhub\api\models\vpn\VpnWhitelistResource.dm"
 #include "code\modules\goonhub\api\requests\apiBody.dm"
+#include "code\modules\goonhub\api\requests\admins\AddGameAdminsBody.dm"
+#include "code\modules\goonhub\api\requests\admins\UpdateGameAdminsBody.dm"
 #include "code\modules\goonhub\api\requests\players\notes\AddPlayerNotesBody.dm"
 #include "code\modules\goonhub\api\routes\apiRoute.dm"
+#include "code\modules\goonhub\api\routes\admins\AddGameAdmins.dm"
+#include "code\modules\goonhub\api\routes\admins\DeleteGameAdmins.dm"
+#include "code\modules\goonhub\api\routes\admins\GetGameAdmins.dm"
+#include "code\modules\goonhub\api\routes\admins\UpdateGameAdmins.dm"
```
Comment (ZeWaka) [inline goonstation.dme:1167]: rename files to `*/gameadmins/*.dm`

**Review by ZeWaka — CHANGES_REQUESTED** [review]

**Pull request merged**